ConvivaTestApp/src/androidTest/java/com/bitmovin/analytics/conviva/testapp/SessionTerminationTests.kt

```diff
@@ -33,8 +32,8 @@ class SessionTerminationTests : TestBase() {
             clearMocks(playerStateManagerMock!!)
             clearMocks(clientMock!!)
             activity.bitmovinPlayer.seek(activity.bitmovinPlayer.duration - 1)
-            Thread.sleep(4000)
         }
+        Thread.sleep(4000)
```

Comment:
Same question as in the other test files. And why do we need 4000 instead of 2000ms here?

Reply:
This cases need to wait for seek to be completed and the playback to be finished so increased sleep to 4 seconds.

Reply:
Can't we wait for the `Seeked` event or so instead to know when seeking completed?

Reply:
I explored this when writing tests. To achieve this the test case will need to register events listeners, then add a mechanism to block execution until the expected event is received and continue execution on receiving the expected event. I did not find a direct way to implement this event based delay/wait mechanism using the test framework.

Reply:
Understood, fine for me then.